### PR TITLE
added feature to upload image to OwnCloud Account

### LINKED ISSUE #834: Setup and Uploading image on OwnCloud Account
**Actual Behaviour**

Setting up and uploading am image on OwnCould Account. 

**Expected Behaviour**

User can login from account manager and can upload from SharingActivity.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by pa1pal**
Fix #834 

Changes: Added the OwnCloud item in sharing activity table item. 
Used the library to upload image (not working right now, might be due to some server error where I hosted account). I will follow this in subswquent PRs

Screenshot: 
screen shot 2017-07-22 at 10 18 13 pm

### CODE REVIEW

File: app/src/main/java/org/fossasia/phimpme/share/SharingActivity.java [hunk 1]
```diff
@@ -880,7 +935,7 @@ void shareToNextCloud(){
             phimpmeProgressBarHandler.show();
 
         } else {
-            Toast.makeText(this, "Please sign in to nextcloud from account manager"
+            Toast.makeText(this, "Please sign in to " + account + " from account manager"
```
Comment (anantprsd5) [inline app/src/main/java/org/fossasia/phimpme/share/SharingActivity.java]: Using a snackbar here would be good I think as we are using it for other accounts.

Comment (pa1pal) [inline app/src/main/java/org/fossasia/phimpme/share/SharingActivity.java]: Done

File: app/src/main/java/org/fossasia/phimpme/share/SharingActivity.java [hunk 2]
```diff
@@ -162,22 +178,46 @@
...
+
+    /**
+     * Collection of dark theme colors for accounts table in sharing activity
+     */
+    private int[] cellcolors_DarkTheme = {R.color.facebook_color_darktheme, R.color.twitter_color_darktheme
+            , R.color.instagram_color_darktheme, R.color.wordpress_color_darktheme
+            , R.color.pinterest_color_darktheme, R.color.flickr_color_darktheme
+            , R.color.nextcloud_color_darktheme, R.color.imgur_color_darktheme
+            , R.color.dropbox_color_darktheme, R.color.other_share_color_darktheme
+            , R.color.owncloud_color, R.color.other_share_color_darktheme};
```
Comment (vinaysajjanapu) [inline app/src/main/java/org/fossasia/phimpme/share/SharingActivity.java]: use a lighter shade color for dark theme like all the other colors. Donot use the same color. Use this #45556c for dark theme.

Comment (pa1pal) [inline app/src/main/java/org/fossasia/phimpme/share/SharingActivity.java]: @vinaysajjanapu Done

**Pull request merged**